src/components/bills/CreateBillForm.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { Formik, Form } from "formik";
+import { isNumeric } from "validator";
+import SubmitButtonSpinner from "components/forms/SubmitSpinnerButton";
+import ErrorMessage from "components/ErrorMessage";
+import Input from "components/forms/Input";
+
+const CreateBillForm = ({
+  onSubmit,
+  isSubmitting,
+  errorMessage,
+  hideError
+}) => {
+  const initialValues = {
+    label: "",
+    billercode: "",
+    reference: "",
+    fiat: null
+  };
+
+  const validate = ({ label, billercode, reference, fiat }) => {
+    const errors = {};
+    const reqMsg = "This field is required";
+    if (!billercode) errors.billercode = reqMsg;
+    if (!reference) errors.ref = reqMsg;
+    if (!isNumeric(String(fiat))) errors.fiat = "Amount must be a number";
+
+    if (label.length > 30)
+      errors.label = "Label must be shorter than 30 characters";
+    if (fiat > 10000000)
+      errors.fiat = "AUD amount must be less than 10.000.000";
```

Comment:
```suggestion
      errors.fiat = "AUD amount must be less than 10,000.000";
```

Reply:
it's 10 millions so `10,000,000` - 2 commas?